pootle/templates/user/profile.html

```diff
@@ -115,12 +30,12 @@ <h3 class="user-position">
 <script type="text/javascript" src="{{ ASSET_URL }}"></script>
 {% endassets %}
 <script type="text/javascript">
-$(function() {
-  PTL.user.init({
-    userData: {{ user.field_values|to_js }},
-    appRoot: '{{ user.get_absolute_url }}',
-  });
-});
+    $(function() {
+        PTL.user.init({
+            userData: {{ user.field_values|to_js }},
+            appRoot: '{{ user.get_absolute_url }}',
+        });
+    });
```

Comment:
Not necessarily, but even having that in account in templates we indent by 2 spaces at a time and here you are using 4 spaces.